xsd/netex_framework/netex_reusableComponents/netex_vehicle_version.xsd

```diff
@@ -179,7 +179,6 @@ Rail transport, Roads and Road transport
 			<xsd:element ref="ContactRef" minOccurs="0"/>
 			<xsd:element ref="TransportTypeRef" minOccurs="0"/>
 			<xsd:element ref="VehicleModelRef" minOccurs="0"/>
-			<xsd:element ref="VehicleModelProfileRef" minOccurs="0"/>
```

Comment:
This is breaking some examples (NeTEx\examples\functions\newModes\NewModes-CarClubExample.xml )

Reply:
@Aurige do you think we should keep it or adapt the exampled?

Reply:
I don't really like breaking example (unless they were obviously wrong)
The **VehicleCodeGroup** is only used once (in **VehicleGroup**): so I would remove the **equipmentProfiles** and **VehicleModelProfileRef** from it and move them to this line 182 (in fact just remove them from **VehicleCodeGroup** and add **equipmentProfiles**  before **VehicleModelProfileRef**  of line 182 ... 
That should fix the issue

Reply:
done